device.mk

```diff
@@ -209,4 +209,26 @@ $(call add-product-dex-preopt-module-config,services,--compiler-filter=speed)
 #
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
     persist.sys.usb.config=mtp \
-    persist.data.qmi.adb_logmask=0
+    persist.data.qmi.adb_logmask=0 \
+    ro.adb.secure=0
+
+# for Gecko to support bluedroid stack
+PRODUCT_PACKAGES += \
+    bluetooth.default
+
+ENABLE_LIBRECOVERY := true
+
+# Enable virtual home button for b2g
+PRODUCT_PROPERTY_OVERRIDES += \
+    ro.moz.has_home_button=0 \
+    ro.moz.nfc.enabled=true \
+    ro.moz.ril.ipv6=true
```

Comment:
Hsin-Yi removed this ipv6 property on Z3/Z3c because the system was not ready yet. I'm not sure Lollipop is either.